src/BODSDataExtractor/extractor.py

```diff
@@ -1686,15 +1686,16 @@ def services_on_bods_or_otc_by_area(self):
         #try except ensures that this reads in lookup file whether pip installing the library, or cloning the repo from GitHub
         try:
             #import the csv file as a text string from the BODSDataExtractor package
-            atco_lookup_file = importlib.resources.read_text('BODSDataExtractor','ATCO_code_to_LA_lookup.csv')
-            
+            atco_lookup_file = importlib.resources.read_text('BODSDataExtractor',
+                                                             'BODSDataExtractor/ATCO_code_to_LA_lookup.csv')
             #wrap lookup_file string into a stringIO object so it can be read by pandas
             atco_lookup_string = io.StringIO(atco_lookup_file)
 
             la_lookup = pd.read_csv(atco_lookup_string ,dtype={'ATCO Code':str})
-            
+            la_lookup['ATCO Code'] = la_lookup['ATCO Code'].astype(str)

```

Comment:
do you need this line considering the above line (1694) looks like it enforces the datatype to be string?